diff --git a/Makefile.libretro b/Makefile.libretro
index 52d690f..9e291b0 100644
--- a/Makefile.libretro
+++ b/Makefile.libretro
@@ -592,6 +592,21 @@ else ifeq ($(platform), miyoo)
 	BUILTIN_GPU = unai
 	DYNAREC = ari64
 
+# RGB30
+else ifeq ($(platform), rgb30)
+   TARGET := $(TARGET_NAME)_libretro.so
+   CC = $(CROSS_COMPILE)gcc
+   CXX = $(CROSS_COMPILE)g++
+   AR = $(CROSS_COMPILE)ar
+   CFLAGS += -mtune=cortex-a55 -march=armv8-a+crc+simd
+   CFLAGS += -fomit-frame-pointer -ffast-math -fdata-sections -ffunction-sections -fsingle-precision-constant -flto -fPIC -ftree-vectorize
+   LDFLAGS += -flto -fPIC
+   OPTIMIZE += -Ofast -DNDEBUG=1
+   HAVE_NEON = 1
+   BUILTIN_GPU = neon
+   ARCH = arm64
+   DYNAREC = ari64
+
 # Emscripten
 else ifeq ($(platform), emscripten)
    TARGET  := $(TARGET_NAME)_libretro_$(platform).bc
